### ISSUE TITLE
Add Softmax Output for DebertaAnswerScores batch_calculate method

### ISSUE THREAD

**Issue by satriowputra** [relari-ai/continuous-eval#6]
In `DebertaAnswerScores`, we need to ensure consistency in the output of the `calculate` and `batch_calculate` methods. In current implementation, the `calculate` method returns normalized probabilities, while the `batch_calculate` return logits values.
![image](https://github.com/relari-ai/continuous-eval/assets/64181224/3eb3018c-8b82-4644-8299-60efb4fb3f69)
